metadata.yaml

```diff
@@ -42,5 +47,13 @@ provides:
     interface: ldap
   glauth-auxiliary:
     description: |
-      Provides auxiliary data for glauth-utils charmed operator
+      Provides auxiliary data for glauth-utils charmed operator.
     interface: glauth_auxiliary
+  certificates-transfer:
```

Comment:
Let's change this to `send-ca-cert`, to describe intent and also to keep it consistent with how it is named in other charms.

Reply:
Yeah, I will make it consistent. But TBH, this integration name sounds confusing to me.